src/services/panthalassa/index.js

```diff
@@ -0,0 +1,318 @@
+ * @returns {Promise<*>} Boolean response about operation's result
+ */
+export async function panthalassaStartDApp(dApp: String, timeout: Number):Promise<boolean> {
+  return Panthalassa.PanthalassaStartDApp({
+    dApp,
+    timeout,
+  });
+}
+
+/**
+ * @desc Opens an specific Dapp
+ * @param {string} id DApp's Id
+ * @param {string} context TODO
+ * @returns {Promise<*>} Boolean response about operation's result
+ */
+export async function panthalassaOpenDApp(id: String, context: String):Promise<boolean> {
+  return Panthalassa.PanthalassaOpenDApp({
+    id,
+    context,
+  });
+}
+
+/**
+ * @desc TODO
+ * @returns {Promise<*>} TODO
+ */
+export async function panthalassaNewPreKeyBundle():Promise<string> {
+  return Panthalassa.PanthalassaNewPreKeyBundle();
+}
+
+/**
+ * @desc Calls a DApp's function over th VM
+ * @param {string} dAppId TODO
+ * @param {string} id TODO
+ * @param {string} args TODO
+ * @returns {Promise<*>} Boolean response about operation's result
+ */
+export async function panthalassaCallDAppFunction(dAppId: String, id: Number, args: String):Promise<boolean> {
+  return Panthalassa.PanthalassaCallDAppFunction({
+    dAppId,
+    id,
+    args,
+  });
+}
+
+/**
+ * @desc TODO
+ * @param {string} address TODO
+ * @returns {Promise<*>} TODO
+ */
+export async function panthalassaConnectToDAppDevHost(address: String):Promise<boolean> {
+  return Panthalassa.PanthalassaConnectToDAppDevHost(address);
+}
+
+/**
+ * @desc TODO
+ * @param {string} rawMsg TODO
+ * @param {string} secretID TODO
+ * @param {string} secret TODO
+ * @param {string} receiverIdKey TODO
+ * @returns {Promise<*>} TODO
+ */
+export async function panthalassaCreateDAppMessage(rawMsg: String, secretID: String, secret: String, receiverIdKey:String):Promise<string> {
+  return Panthalassa.PanthalassaCreateDAppMessage({
+    rawMsg,
+    secretID,
+    secret,
+    receiverIdKey,
+  });
+}
+
+/**
+ * @desc TODO
+ * @param {string} identityPublicKey TODO
+ * @param {string} preKeyBundle TODO
+ * @returns {Promise<*>} {string} TODO
+ */
+export async function panthalassaInitializeChat(identityPublicKey: String, preKeyBundle: String):Promise<string> {
+  return Panthalassa.PanthalassaInitializeChat({
+    identityPublicKey,
+    preKeyBundle,
+  });
+}
+
+/**
+ * @desc TODO
+ * @param {string} message TODO
+ * @param {string} preKeyBundlePrivatePart TODO
+ * @returns {Promise<*>} {string} TODO
+ */
+export async function panthalassaHandleInitialMessage(message: String, preKeyBundlePrivatePart: String):Promise<string> {
+  return Panthalassa.PanthalassaHandleInitialMessage({
+    message,
+    preKeyBundlePrivatePart,
+  });
+}
+
+/**
+ * @desc TODO
+ * @param {string} rawMsg TODO
+ * @param {string} secretID TODO
+ * @param {string} secret TODO
+ * @param {string} receiverIdKey TODO
+ * @returns {Promise<*>} {string}
+ */
+export async function panthalassaCreateHumanMessage(rawMsg: String, secretID: String, secret: String, receiverIdKey: String):Promise<string> {
```

Comment:
I don't think it's needed, since chat is moved to Panthalassa.

Reply:
Deleted